Assets/scss/components/article-cards.scss

```diff
@@ -0,0 +1,95 @@
+.article-cards{
+    background-color: $white-smoke;
+    font-family: $arial;
+    max-width: 100%;
+    margin: 0;
+
+    &__menu{
+        display: flex;
+        padding: 20px;
+       
+        &--item{
+            @include article-dropdown-style;
+            max-width: 150px;
+        }
+        .dropdown-up{
+            @include article-dropdown-style;
+        }
+    }
+
+    &__container{
+        max-width: inherit;
+        display: grid;
+        gap: 20px;
+        @include min-screen(sm) {
+            padding: 0px 20px;
+        }
+        @include min-screen(md) {
+            grid-template-columns: repeat(2, 1fr);
+
+        }
+        @include min-screen(lg) {
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+    &__card{
+        background-color: var(--card-bg-color);
+        margin-bottom: 20px;
+        &--image{
+            width: 770px;
+        }
+        &--detailed{
+            display: flex;
+            margin-left: 1rem;
+            margin-top: 1rem;
+            .card-detailed__location{
+                font-weight: $font-bold;
+                color: $wine;
+
+                &::after{
+                    content: "   | ";
+                    padding: 0px 8px;
+                }
+
+            }
+            .card--detailed__publishdate{
+                font-size: 12px;
```

Comment:
Could use `toRem(12)` function for all font-sizes

Reply:
Code updated and used toRem().